records_mover/db/redshift/loader.py

```diff
@@ -94,4 +94,28 @@ def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool
             return False
 
     def known_supported_records_formats_for_load(self) -> List[BaseRecordsFormat]:
-        return [DelimitedRecordsFormat(variant='bluelabs')]
+        return [
+            # Redshift is pretty flexible with date/time/timezone
+            # parsing, so let's use them and go back to saner defaults
+            # than those of MS Excel - e.g., the default csv and bigquery
+            # formats don't include timezones.
+            DelimitedRecordsFormat(variant='csv',
+                                   hints={
+                                       'dateformat': 'YYYY-MM-DD',
+                                       'timeonlyformat': 'HH24:MI:SS',
+                                       'datetimeformat': 'YYYY-MM-DD HH:MI:SS',
+                                       'datetimeformattz': 'YYYY-MM-DD HH:MI:SSOF',
+                                   }),
+            # 'bigquery' and 'csv' support both newlines in strings as
+            # well as empty strings (but not timezones)
+            DelimitedRecordsFormat(variant='bigquery'),
+            DelimitedRecordsFormat(variant='csv'),
+            # The default 'bluelabs' format can't represent empty
+            # strings - but it can with this flag.  Unfortunately, it
+            # doesn't support newlines in strings.
+            DelimitedRecordsFormat(variant='bluelabs', hints={
+                'quoting': 'all',
+            }),
+            # Supports newlines in strings, but not empty strings.
+            DelimitedRecordsFormat(variant='bluelabs'),
```

Comment:
We could potentially create some additional supported types - it might also be a reasonable time to rename 'bluelabs' (maybe 'escaped'?) and then publish the spec somewhere public...